### PR TITLE
257 update drls landing page

### PR DESCRIPTION
**Pull request by mulcahyk**

### CODE REVIEW

File: server/src/main/resources/templates/index.html [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <title>React App</title>
     <link href="main.index.css" rel="stylesheet">
     <meta name="ctx" th:content="${contextPath}" />
+    <meta name="hostorg" th:content="${hostOrg}" />
```
Comment (mulcahyk) [inline server/src/main/resources/templates/index.html]: @KeeyanGhoreshi  does this need to stay or is it just something for testing locally?

Comment (KeeyanGhoreshi) [reply server/src/main/resources/templates/index.html]: When the server is deployed it's going to get the `hostOrg` from this index.html, not the one in `node_interfaces`.

File: server/src/main/resources/application.yml [hunk 2]
```diff
@@ -31,4 +31,5 @@ cdsConnect:
   password: admin
 
 localDbRules: src/main/jib/rules/
-localDbFhirArtifacts: src/main/jib/smartAppFhirArtifacts/
```
Comment (KeeyanGhoreshi) [inline server/src/main/resources/application.yml]: Did you mean to delete this?

Comment (mulcahyk) [reply server/src/main/resources/application.yml]: No.  I saw this line get dropped when I did a rebase against master at one point.  I assume that it should be added back in then?

Comment (mulcahyk) [reply server/src/main/resources/application.yml]: Added it back.

**Comment by lpdecelles** [comment]
Kristian, It looks like there are still spacing issues.

**Pull request merged**